examples/libs/algorand/asset-tx/main.ts

```diff
@@ -0,0 +1,91 @@
+import { DfnsWallet, singleSigTransactionToEncoded } from '@dfns/lib-algorand'
+import { DfnsApiClient } from '@dfns/sdk'
+import { AsymmetricKeySigner } from '@dfns/sdk-keysigner'
+import { Algodv2, encodeObj, makeAssetTransferTxnWithSuggestedParams, waitForConfirmation } from 'algosdk'
+
+import * as dotenv from 'dotenv'
+
+dotenv.config()
+
+const initDfnsWallet = async (walletId: string) => {
+  const signer = new AsymmetricKeySigner({
+    privateKey: process.env.DFNS_PRIVATE_KEY!,
+    credId: process.env.DFNS_CRED_ID!,
+    appOrigin: process.env.DFNS_APP_ORIGIN!,
+  })
+
+  const dfnsClient = new DfnsApiClient({
+    appId: process.env.DFNS_APP_ID!,
+    authToken: process.env.DFNS_AUTH_TOKEN!,
+    baseUrl: process.env.DFNS_API_URL!,
+    signer,
+  })
+
+  return DfnsWallet.init({
+    walletId: walletId,
+    dfnsClient,
+  })
+}
+
+const TESTNET_USDC_ASSET_ID = 10458941
+
+async function main() {
+  const receiverWallet = await initDfnsWallet(process.env.ALGORAND_RECEIVER_WALLET_ID!)
+  console.log('algorand receiver address: %s', receiverWallet.address)
+
+  const senderWallet = await initDfnsWallet(process.env.ALGORAND_SENDER_WALLET_ID!)
+  console.log('algorand sender address: %s', senderWallet.address)
+
+  const algod = new Algodv2(process.env.ALGORAND_NODE_API_KEY!, process.env.ALGORAND_NODE_URL!)
```

Comment:
if you make this global const, then don't have to pass it around to all the helpers.